spec/factories/orders.rb

```diff
@@ -0,0 +1,7 @@
+FactoryBot.define do
+  factory :order do
+    product_name { "MyString" }
+    product_count { 1 }
```

Comment:
[This section](https://github.com/IrynaKolh/R7-additional/blob/main/lessons/Assignment-10-Rails-Comprehensive-Assignment.md#a-few-new-ideas) of the assignment provides a way to generate different names and counts, so that every order created doesn't have exactly the same values (other than ID). However, the tests still work; they just don't look as "natural."